CMakeLists.txt

```diff
@@ -12,7 +12,11 @@ option(USE_VCPKG "Use VCPKG to download and manage dependencies" OFF)
 
 if (USE_VCPKG)
     # Detect correct triplet based on CMake env
-    if (WIN32)
+    if (CMAKE_SYSTEM_NAME STREQUAL "Mingw")
+        set(VCPKG_TARGET_TRIPLET "x64-mingw-static")
+        set(VCPKG_INSTALL_OPTIONS "--allow-unsupported")
+        set(PTHREAD_LIBRARY "-lpthread")
```

Comment:
Why are these needed? I don't think they should be

Reply:
vcpkg gives the following error message if `--allow-unsupported` is not specified when cross-compiling:
```
vcpkg-msbuild is only supported on 'native & windows', which does not match x64-linux. This usually means that there are known build failures, or runtime problems, when building other platforms. To ignore this and attempt to build vcpkg-msbuild anyway, rerun vcpkg with `--allow-unsupported`.
```